Examples/MAX78000/EEPROM_Emulator/README.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+This example utilizes the MAX78000 to emulate a 32KiB EEPROM chip.
+
+This "EEEPROM" can only perform read and write operations.
```

Comment:
Consider linking to the I2C spec.  Unfortunately the official spec is behind an NXP account wall, but this link has some public info:  https://i2c.info/i2c-bus-specification